MovingBlocks/TerasologyLauncher
# Remove G+ icon :( and add Discord icon instead :)

### Contains

Changes the third leftmost icon in the bottom bar to a far more relevant website :)
Icon is my own, based on the official logo from https://discordapp.com/branding.

### How to test

Launch the launcher, then verify that the third button in the sidebar no longer launches a (dead) gplus link, but rather https://discord.gg/terasology.

### Outstanding before merging

All good!
